crates/pkgcruft/src/test.rs

```diff
@@ -23,6 +23,25 @@ impl ToReports for Command {
     }
 }
 
+#[cfg(test)]
```

Comment:
It feels wrong to move this from the end of the file as that doesn't match where other unit tests are placed in modules.

Reply:
it was a leftover from moving those tests in few places, fixed.